u_parrots.go

```diff
@@ -350,43 +804,49 @@ func utlsIdToSpec(id ClientHelloID) (ClientHelloSpec, error) {
 				TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,
 				TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
 				TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
-				FAKE_TLS_DHE_RSA_WITH_AES_128_CBC_SHA,
-				FAKE_TLS_DHE_RSA_WITH_AES_256_CBC_SHA,
+				TLS_RSA_WITH_AES_128_GCM_SHA256,
+				TLS_RSA_WITH_AES_256_GCM_SHA384,
 				TLS_RSA_WITH_AES_128_CBC_SHA,
 				TLS_RSA_WITH_AES_256_CBC_SHA,
-				TLS_RSA_WITH_3DES_EDE_CBC_SHA,
 			},
 			CompressionMethods: []byte{
 				compressionNone,
 			},
 			Extensions: []TLSExtension{
-				&SNIExtension{},
-				&UtlsExtendedMasterSecretExtension{},
-				&RenegotiationInfoExtension{Renegotiation: RenegotiateOnceAsClient},
-				&SupportedCurvesExtension{[]CurveID{
+				&SNIExtension{},                      //server_name
+				&UtlsExtendedMasterSecretExtension{}, //extended_master_secret
+				&RenegotiationInfoExtension{Renegotiation: RenegotiateOnceAsClient}, //extensionRenegotiationInfo
+				&SupportedCurvesExtension{[]CurveID{ //supported_groups
 					X25519,
 					CurveP256,
 					CurveP384,
 					CurveP521,
 					CurveID(FakeFFDHE2048),
 					CurveID(FakeFFDHE3072),
 				}},
-				&SupportedPointsExtension{SupportedPoints: []byte{
+				&SupportedPointsExtension{SupportedPoints: []byte{ //ec_point_formats
 					pointFormatUncompressed,
 				}},
 				&SessionTicketExtension{},
-				&ALPNExtension{AlpnProtocols: []string{"h2", "http/1.1"}},
+				&ALPNExtension{AlpnProtocols: []string{"h2"}}, //application_layer_protocol_negotiation
```

Comment:
Interesting. According to TLSfingerprint.io, the current fingerprint with only h2 is also a fingerprint of Firefox 102. 

But I agree that we may want to move toward one with better compatibility.

Reply:
On a second thought, I decide not to change it. Since we have Firefox 105 which is essentially [b250617dca0ac79f](https://tlsfingerprint.io/id/b250617dca0ac79f), we want to preserve the variety.